### DISCUSSION TITLE
Azure Virtual Network Manager add-on for reference implementations

### DISCUSSION THREAD

**Discussion by pazdedav** [Azure/Enterprise-Scale#849] [Ideas] [answered]
At Ignite, a Public Preview of **Azure Virtual Network Manager** was announced.

Has there been any considerations in the core team to add this functionality to reference implementations and replace more traditional peering and distributed NSG management with AVNM configurations?

_Some of my customers are struggling to manage NSGs (and ASGs) at scale, especially since they cannot be shared across subscriptions._

**Comment by jtracey93** [comment, marked as answer]
This indeed is on our backlog 👍

However, as its only just been announced our guidance will likely follow once we are closer to GA for AVNM

**Reply by pazdedav** [reply]
Thanks, @jtracey93. The availability of AVNM is indeed limited in the Preview (features and regions) but it's great to hear this could be a direction ALZ guidance and RIs are going :)
